package.json

```diff
@@ -14,8 +14,9 @@
     "stylelint-scss": "^3.17.2"
   },
   "devDependencies": {
-    "webpack": "^5.11.0",
-    "webpack-cli": "^4.3.0"
+    "webpack": "^5.11.1",
+    "webpack-cli": "^4.3.0",
+    "webpack-dev-server": "^3.11.1"
```

Comment:
- [ ] I still got an error when I start the `start` script. `webpack-cli` comes with `serve` to start the `webpack-dev-server` so we can replace our `start` script to:

```json
"start": "webpack serve"
```
Kindly take a look at this [issue](https://github.com/webpack/webpack-dev-server/issues/2759) on GitHub.

By default, webpack server will server your root directory, instead we want to serve only the `dist` folder. This can be done with adding webpack server configurations in `webpack.config.js`

```js
devServer: {
  contentBase: path.join(__dirname, 'dist'),
  port: 3000,
},
```

Kindly take a look other [configurations options](https://webpack.js.org/configuration/dev-server/)

Reply:
This has been implemented, thanks!